.github/workflows/wheels.yml

```diff
@@ -31,8 +31,6 @@ jobs:
                 exclude:
                   - os: windows-2019
                     arch: "x86_64"
-                  - os: ubuntu-20.04
-                    arch: "arm64"
```

Comment:
This is excluded because this combination doesn't exist, the correct one is ubuntu-20.04/aarch64 and is already included above.